fdbclient/NativeAPI.actor.cpp

```diff
@@ -242,7 +242,7 @@ void DatabaseContext::getLatestCommitVersions(const Reference<LocationInfo>& loc
 		return;
 	}
 
-	if (ssVersionVectorCache.getMaxVersion() != invalidVersion && readVersion > ssVersionVectorCache.getMaxVersion()) {
+	if (readVersion > ssVersionVectorCache.getMaxVersion()) {
```

Comment:
this check was redundant per line 241 above.

Reply:
This was my bad, actually. I should have removed this redundant check as part of commit c74710f6d5.